zwave_js_server/model/node.py

```diff
@@ -156,34 +163,44 @@ def is_listening(self) -> Optional[bool]:
         return self.data.get("isListening")
 
     @property
-    def is_frequent_listening(self) -> Optional[bool]:
+    def is_frequent_listening(self) -> Optional[Union[bool, str]]:
         """Return the is_frequent_listening."""
-        return self.data["isFrequentListening"]
+        return self.data.get("isFrequentListening")
 
     @property
     def is_routing(self) -> Optional[bool]:
         """Return the is_routing."""
         return self.data.get("isRouting")
 
     @property
-    def max_baud_rate(self) -> Optional[int]:
-        """Return the max_baud_rate."""
-        return self.data.get("maxBaudRate")
+    def max_data_rate(self) -> Optional[int]:
+        """Return the max_data_rate."""
+        return self.data.get("maxDataRate")
+
+    @property
+    def supported_data_rates(self) -> Optional[List[int]]:
+        """Return the supported_data_rates."""
+        return self.data.get("supportedDataRates")
```

Comment:
Containers should always default to empty container.
```suggestion
        return self.data.get("supportedDataRates", [])
```